package.json

```diff
@@ -0,0 +1,5 @@
+{
+  "scripts": {
+    "preinstall": "cd website && npm install"
```

Comment:
You could add a "start" script that starts the the website in /website

Reply:
In the `website` package json add a `"postinstall": "npm run build"` so that it builds the react app after installing. This let's us avoid the impact of having to build the app in every cold start, it is built on every deploy.